### PR TITLE
Fixes unsubscribe on multiple subscribers after destroy

### LINKED ISSUE #31: Proppy subscribers are non removed properly
Hey Fahad! Great job on Proppy, I've been quite amazed by the project so far but ran into a small bug:

The implementation of `removeCallback` will not remove all handlers when .destroy is called, you can see an example here:

https://codesandbox.io/s/o499mj9nxy?expanddevtools=1&module=%2Fsrc%2Findex.js

### PR DESCRIPTION
**Pull request by reaktivo**
Fixes #31

### CODE REVIEW

**Comment by fahad19** [comment]
The build failed at `lerna bootstrap` for some reason. I have restarted it on Travis-CI.

File: packages/proppy/src/create.ts [hunk 1]
```diff
@@ -91,7 +80,7 @@ export function create(options: ProppyFactoryOptions): ProppyFactory {
         parent.destroy();
       }
 
-      removeCallback(callbacks);
+      callbacks = []
```
Comment (fahad19) [inline packages/proppy/src/create.ts]: linting is failing because of missing semicolon 😅

**Comment by fahad19** [comment]
@reaktivo: upgraded to TypeScript v3, and also dropped a package in `master`. builds are now passing for `master`.

Merging latest `master` in your branch should now pass you build.

Thanks for your patience! 🙌

**Comment by reaktivo** [comment]
Rebased on master and checks passed!

**Comment by fahad19** [comment]
Thanks @reaktivo! Landed in v1.2.5: https://github.com/fahad19/proppy/releases/tag/v1.2.5

**Pull request merged**